src/linear_algebra/char_poly/coeff.lean

```diff
@@ -85,7 +85,8 @@ by { rw fintype.card_eq_zero_iff at h, suffices : M = 1, { simp [this] }, ext, t
 theorem char_poly_degree_eq_dim [nontrivial R] (M : matrix n n R) :
 (char_poly M).degree = fintype.card n :=
 begin
-  by_cases fintype.card n = 0, rw h, unfold char_poly, rw det_of_card_zero, simpa,
+  by_cases fintype.card n = 0,
+  { rw h, unfold char_poly, rw det_of_card_zero, simp, assumption },
```

Comment:
```suggestion
  { rw h, unfold char_poly, rw det_of_card_zero, {simp}, {assumption} },
```
Although it looks like the proofs in this file don't really follow the rule to surround new goals in braces at all...